client_cli/src/main.rs

```diff
@@ -771,12 +770,42 @@ mod peer {
                 key,
                 metadata: Metadata(metadata),
             } = self;
+            let unregister_peer: Instruction =
+                UnregisterBox::new(IdBox::PeerId(PeerId::new(&address, &key))).into();
+            submit([unregister_peer].into(), cfg, metadata).wrap_err("Failed to unregister peer")
+        }
+    }
+}
+
+mod wasm {
+    use std::{fs::OpenOptions, io::Read, path::PathBuf};
+
+    use super::*;
+
+    /// Subcommand for dealing with wasm
+    #[derive(Debug, StructOpt)]
+    pub struct Args {
+        /// Path to the wasm file
+        path: PathBuf,
```

Comment:
What about accepting wasm from `stdin`?

Reply:
Added ability to read from `stdin`.

Reply:
This seems superfluous, but if it's implemented let's keep it for the time being.